kubectl-fdb/cmd/root.go

```diff
@@ -145,3 +149,19 @@ func printStatement(cmd *cobra.Command, line string, mesType messageType) {
 	cmd.Printf("✔ %s\n", line)
 	color.Unset()
 }
+
+// will check plugin version and won't let any interaction happen with cluster, if it's not latest release version
+func usingLatestPluginVersion() bool {
+	if latestPluginVersion == "" {
+		return false
+	}
+	isLatest := strings.Compare(strings.ToLower(pluginVersion), "latest") == 0 || strings.Compare(pluginVersion, latestPluginVersion) >= 0
+	if !isLatest {
+		versionMessage = "Your kubectl-fdb plugin is not up-to-date, please install latest version and try again!\n" +
+			"Your version:[" + pluginVersion + "] vs. latest release version:[" + latestPluginVersion + "]\n" +
+			"Installation instructions can be found here: https://github.com/bktsh/fdb-kubernetes-operator/blob/main/kubectl-fdb/Readme.md"
+		fmt.Println(versionMessage)
```

Comment:
```suggestion
		cmd.Println(versionMessage)
```

When we make use of the `PreRunE` method we can make use of the cmd printer. That will print out the message to the according out writer (will be easier to test in unit tests).